frontend/packages/console-dynamic-plugin-sdk/src/api/common-types.ts

```diff
@@ -91,6 +91,7 @@ export type MatchLabels = {
 export type Selector = {
   matchLabels?: MatchLabels;
   matchExpressions?: MatchExpression[];
+  filterByName?: string;
```

Comment:
`filterByName` is custom method in tekton-results utils, so you could extend this Selector type to add filterByName prop on the place where you need it instead of modifying the sdk common-types (mainly applicable for K8s selector).

Reply:
I updated the PR and removed `filterByName`, before I was explicitly using `filterByName` since I moved to hook, I am not using it anymore.